public/js/submit.js

```diff
@@ -137,59 +49,69 @@ function sendData(data, url)
   var urlEncodedDataPairs = [];
   var name;
 
-  // Transformez l'objet data en un tableau de paires clé/valeur codées URL.
+  // Transformez l"objet data en un tableau de paires clé/valeur codées URL.
   for(name in data) {
-    urlEncodedDataPairs.push(encodeURIComponent(name) + '=' + encodeURIComponent(data[name]));
+    urlEncodedDataPairs.push(encodeURIComponent(name) + "=" + encodeURIComponent(data[name]));
   }
 
   // Combinez les paires en une seule chaîne de caractères et remplacez tous
-  // les espaces codés en % par le caractère'+' ; cela correspond au comportement
+  // les espaces codés en % par le caractère"+" ; cela correspond au comportement
   // des soumissions de formulaires de navigateur.
-  urlEncodedData = urlEncodedDataPairs.join('&').replace(/%20/g, '+');
+  urlEncodedData = urlEncodedDataPairs.join("&").replace(/%20/g, "+");
 
   // Définissez ce qui se passe en cas de succès de soumission de données
-  // XHR.addEventListener('load', function(event) {
-  //   alert('Ouais ! Données envoyées et réponse chargée.');
+  // XHR.addEventListener("load", function(event) {
+  //   alert("Ouais ! Données envoyées et réponse chargée.");
   // });
 
-  // Définissez ce qui arrive en cas d'erreur
-  XHR.addEventListener('error', function(event) {
-    alert('Oups! Une erreur s\' produite,  avec l\'object XMLHttpRequest permettant l\'interraction entre serveurs.');
+  // Définissez ce qui arrive en cas d"erreur
+  XHR.addEventListener("error", function(event) {
+    alert("Oups! Une erreur s\" produite,  avec l\"object XMLHttpRequest permettant l\"interraction entre serveurs.");
   });
 
   // Configurez la requête
-  // XHR.open('POST', 'https://example.com/cors.php');
-  XHR.open('POST', url);
-  // XHR.open('POST', 'sendMessage');
+  // XHR.open("POST", "https://example.com/cors.php");
+  XHR.open("POST", url);
+  // XHR.open("POST", "sendMessage");
   
-  // Ajoutez l'en-tête HTTP requise pour requêtes POST de données de formulaire
-  XHR.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+  // Ajoutez l"en-tête HTTP requise pour requêtes POST de données de formulaire
+  XHR.setRequestHeader("Content-Type", "application/x-www-form-urlencoded");
 
   // Finalement, envoyez les données.
   // XHR.onreadystatechange = function() 
   XHR.onreadystatechange = function() 
   {//Call a function when the state changes.
+    if(typeof hideSomethingSpecific === 'function'){  
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/soutenance14/ST/pullRequest?prid=8161867)
```suggestion
    if(typeof hideSomethingSpecific === "function"){  
```